examples/nodejs/token-vending-machine/infrastructure/lib/token-vending-machine-stack.ts

```diff
@@ -0,0 +1,51 @@
+import * as path from 'path';
+import * as cdk from 'aws-cdk-lib';
+import {Construct} from 'constructs';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as lambdaNodejs from 'aws-cdk-lib/aws-lambda-nodejs';
+import * as apig from 'aws-cdk-lib/aws-apigateway';
+import * as secrets from 'aws-cdk-lib/aws-secretsmanager';
+
+export class TokenVendingMachineStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const momentoAuthTokenParam = new cdk.CfnParameter(this, 'MomentoAuthTokenParam', {
+      type: 'String',
+      description:
+        'The Momento Auth Token that will be used to vend browser tokens. Will be stored in Secrets Manager.',
```

Comment:
Wording sounds. bit off here?

```suggestion
        'The Momento Auth Token that will be used to vend browser tokens. Generated tokens will be stored in Secrets Manager for ongoing access later.',
```

Reply:
your change isn't accurate - the generated tokens are not stored anywhere. just the original token that you provide as a stack param. happy to tweak the wording if it's not clear, if you have another suggestion?

Reply:
I see... so maybe something like this then?
```suggestion
        'The Momento Auth Token that will be used to vend browser tokens. Generated temporary tokens are derived from a Momento Auth token stored in Secrets Manager.',
```